### PR TITLE
#27500 PHPUnit 8 for module Cookie

### PR DESCRIPTION
**Pull request by lbajsarowicz**
### Description (*)
Preparing PHPUnit Tests to run with PHPUnit 8.
PRs for each module is separated to make tests easier to merge.

### Fixed Issues (if relevant)
1. magento/magento2#27500

### Manual testing scenarios (*)
N/A

### Questions or comments
PR #27701 needs to be merged first (otherwise PHPUnit tests fail)

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

File: app/code/Magento/Cookie/Test/Unit/Block/RequireCookieTest.php [hunk 1]
```diff
@@ -8,34 +8,36 @@
 
 namespace Magento\Cookie\Test\Unit\Block;
 
+use PHPUnit\Framework\TestCase;
+use PHPUnit\Framework\MockObject\MockObject;
```
Comment (eduard13) [inline app/code/Magento/Cookie/Test/Unit/Block/RequireCookieTest.php:12]: Could you please sort these imports ascending?

File: app/code/Magento/Cookie/Test/Unit/Controller/Index/NoCookiesTest.php [hunk 2]
```diff
@@ -5,60 +5,68 @@
  */
 namespace Magento\Cookie\Test\Unit\Controller\Index;
 
+use PHPUnit\Framework\TestCase;
```
Comment (eduard13) [inline app/code/Magento/Cookie/Test/Unit/Controller/Index/NoCookiesTest.php:8]: The same here

File: app/code/Magento/Cookie/Test/Unit/Helper/CookieTest.php [hunk 3]
```diff
@@ -5,39 +5,49 @@
  */
 namespace Magento\Cookie\Test\Unit\Helper;
 
-class CookieTest extends \PHPUnit\Framework\TestCase
+use PHPUnit\Framework\TestCase;
```
Comment (eduard13) [inline app/code/Magento/Cookie/Test/Unit/Helper/CookieTest.php:8]: Same here

File: app/code/Magento/Cookie/Test/Unit/Model/Config/Backend/DomainTest.php [hunk 4]
```diff
@@ -5,29 +5,36 @@
  */
 namespace Magento\Cookie\Test\Unit\Model\Config\Backend;
 
+use PHPUnit\Framework\TestCase;
```
Comment (eduard13) [inline app/code/Magento/Cookie/Test/Unit/Model/Config/Backend/DomainTest.php:8]: Same here

File: app/code/Magento/Cookie/Test/Unit/Model/Config/Backend/LifetimeTest.php [hunk 5]
```diff
@@ -8,33 +8,37 @@
 
 namespace Magento\Cookie\Test\Unit\Model\Config\Backend;
 
+use PHPUnit\Framework\TestCase;
```
Comment (eduard13) [inline app/code/Magento/Cookie/Test/Unit/Model/Config/Backend/LifetimeTest.php:11]: Same here

File: app/code/Magento/Cookie/Test/Unit/Model/Config/Backend/PathTest.php [hunk 6]
```diff
@@ -8,34 +8,38 @@
 
 namespace Magento\Cookie\Test\Unit\Model\Config\Backend;
 
+use PHPUnit\Framework\TestCase;
```
Comment (eduard13) [inline app/code/Magento/Cookie/Test/Unit/Model/Config/Backend/PathTest.php:11]: Same here

File: app/code/Magento/Cookie/Test/Unit/Controller/Index/NoCookiesTest.php [hunk 7]
```diff
@@ -5,60 +5,68 @@
  */
```
Comment (eduard13) [inline app/code/Magento/Cookie/Test/Unit/Controller/Index/NoCookiesTest.php:5]: Could you please also declare the `declare(strict_types=1);`?
```suggestion
 */

declare(strict_types=1);
```

File: app/code/Magento/Cookie/Test/Unit/Helper/CookieTest.php [hunk 8]
```diff
@@ -5,39 +5,49 @@
  */
```
Comment (eduard13) [inline app/code/Magento/Cookie/Test/Unit/Helper/CookieTest.php:5]:
```suggestion
 */

 declare(strict_types=1);
```

File: app/code/Magento/Cookie/Test/Unit/Model/Config/Backend/DomainTest.php [hunk 9]
```diff
@@ -5,29 +5,36 @@
  */
```
Comment (eduard13) [inline app/code/Magento/Cookie/Test/Unit/Model/Config/Backend/DomainTest.php:5]:
```suggestion
 */

declare(strict_types=1);
```

File: app/code/Magento/Cookie/Test/Unit/Model/Config/Backend/LifetimeTest.php [hunk 10]
```diff
@@ -8,33 +8,37 @@
```
Comment (eduard13) [inline app/code/Magento/Cookie/Test/Unit/Model/Config/Backend/LifetimeTest.php:8]:
```suggestion

declare(strict_types=1);

```

File: app/code/Magento/Cookie/Test/Unit/Model/Config/Backend/PathTest.php [hunk 11]
```diff
@@ -8,34 +8,38 @@
```
Comment (eduard13) [inline app/code/Magento/Cookie/Test/Unit/Model/Config/Backend/PathTest.php:8]:
```suggestion

declare(strict_types=1);

```

**Review by eduard13 — CHANGES_REQUESTED** [review]
Hi @lbajsarowicz thank you for these improvements. Could you please check these minor adjustments?
Let's also see the checks result.
Thank you.

**Comment by srsathish92** [comment]
@eduard13 @lbajsarowicz Already PR https://github.com/magento/magento2/pull/27658 raised for Module_Cookie with PHPUnit 8 compatibility. So I think this PR needs to be closed.

**Comment by eduard13** [comment]
Thank you @srsathish92.
@lbajsarowicz closing as a duplicate.
Thank you.

**Pull request closed without merging**